### DISCUSSION TITLE
有人跟我一样出了这个成就吗？

### DISCUSSION THREAD

**Discussion by xingyuehao** [share121/inter-knot#3206] [其他]
洗个脸吧：哈士奇那连出两个骨头。
![屏幕截图 2024-08-19 234546](https://github.com/user-attachments/assets/3b042444-6cf2-41ee-864a-08c8ff9b958c)
<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" >

**Comment by I-ikun** [comment]
我出过<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >
